Makefile

```diff
@@ -109,6 +109,9 @@ test-docker-py: build ## run the docker-py tests
 test-integration-cli: build ## run the integration tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary test-integration-cli
 
+test-splunk-logger: build ## run the logger tests
```

Comment:
I would rather rename this to something more generic that allows for the testing of all loggers in the future.  

Maybe something like `test-log-drivers`? maybe you have a better idea of a generic name, i'm not totally sold on that one